docs/devel/virtual-machine.md

```diff
@@ -142,7 +142,7 @@ The file specification follows the Kubernetes guide. The apiVersion is linked
 with the KubeVirt release cycle.
 
 In the metadata section, there is a *required* field, the **name**. Then
-following the spec section, there are two important parts. The **running**, which
+following the spec section, there are two important parts. The **runStrategy**, which
```

Comment:
I am not sure of this sentence. To me, the runStrategy doesn't indicate the current state of the VMI. :)

Reply:
Changed to "indicates the desired state.." :)
PTAL

Reply:
Thank you! lgtm!